hslua-module-system.cabal

```diff
@@ -33,6 +33,7 @@ library
                      , exceptions >= 0.8    && < 0.11
                      , hslua      >= 1.0.3  && < 1.2
                      , temporary  >= 1.2    && < 1.4
+                     , filepath >= 1.4
```

Comment:
Version might be reviewed here...

Reply:
Using `filepath   > 1.4    && < 1.5` will guarantee that there won't be unexpected breaking changes when filepath is updated. Supporting older filepath versions is not necessary, version 1.3 is more than 5 years old.